domains/_vercel.youssef.json

```
@@ -0,0 +1,9 @@
+{
+"owner": {
+          "username": "YoussefDevPro",
+          "email": "youssef.pro.coding@gmail.com"
+},
+"record": {
+          "TXT": "vc-domain-verify=youssef.is-a.dev,8a21ff17b71b3518be3c"
+}
```

beautified json looks better
```suggestion
  "owner": {
    "username": "YoussefDevPro",
    "email": "youssef.pro.coding@gmail.com"
  },
  "record": {
    "TXT": "vc-domain-verify=youssef.is-a.dev,8a21ff17b71b3518be3c"
  }
```

---

noice